### PR TITLE
optimize fedlearner-add-on

### PR DESCRIPTION
**Pull request by DeliangFan**
Optimize fedleaner add on to make it simpler.

helm install fedlearner-add-on ./fedlearner-add-on \
  --set imageCredentials.registry=xxxx \
  --set imageCredentials.username=xxxx \
  --set imageCredentials.password=xxxx \
  --set service.externalName=xxxx

### CODE REVIEW

**Comment by DeliangFan** [comment]
Test by helm template for compatibility.

File: deploy/charts/fedlearner-add-on/values.yaml [hunk 1]
```diff
@@ -3,32 +3,35 @@
...
 tls:
-  enabled: false
+  enabled: true
   server:
-    key: ""
-    cert: ""
-    intermediate: ""
-    ca: ""
+    key: certs/server/server.key
+    cert: certs/server/server.crt
```
Comment (piiswrong) [inline deploy/charts/fedlearner-add-on/values.yaml:30]: 这个好像是.pem不是.crt

Comment (DeliangFan) [reply deploy/charts/fedlearner-add-on/values.yaml:30]: The certs assigned by dongyu is readly pem format.

File: deploy/charts/fedlearner-add-on/values.yaml [hunk 2]
```diff
@@ -3,32 +3,35 @@
...
   type: ExternalName
   externalName: ""
 
 ingress:
-  enabled: false
+  enabled: true
   annotations:
+    "nginx.ingress.kubernetes.io/proxy-body-size": 10g
+    "nginx.ingress.kubernetes.io/backend-protocol": GRPCS
+    "nginx.ingress.kubernetes.io/http2-insecure-port": t
```
Comment (peng09) [inline deploy/charts/fedlearner-add-on/values.yaml:21]: change t to true, t is a workaround to pass params to yaml

Comment (DeliangFan) [reply deploy/charts/fedlearner-add-on/values.yaml]: fixed

**Pull request merged**